### ISSUE TITLE
Error occured in spigot 1.10.2

### ISSUE THREAD

**Issue by AndyYu168** [EndlessCodeGroup/RPGInventory#75]
My server is spigot 1.10.2 and I'm using RPGInventory 2.1.3 (the latest)
When I jump in game, this error occurs on my console.
Please help me...

```
[18:30:32] [Server thread/ERROR]: Could not pass event PlayerMoveEvent to RPGInventory v2.1.3
org.bukkit.event.EventException
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:308) ~[spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-455072c-43ab266]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-455072c-43ab266]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-455072c-43ab266]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-455072c-43ab266]
	at net.minecraft.server.v1_10_R1.PlayerConnection.a(PlayerConnection.java:610) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-455072c-43ab266]
	at net.minecraft.server.v1_10_R1.PacketPlayInFlying.a(SourceFile:126) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-455072c-43ab266]
	at net.minecraft.server.v1_10_R1.PacketPlayInFlying$PacketPlayInPositionLook.a(SourceFile:18) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-455072c-43ab266]
	at net.minecraft.server.v1_10_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-455072c-43ab266]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_144]
	at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_144]
	at net.minecraft.server.v1_10_R1.SystemUtils.a(SourceFile:45) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-455072c-43ab266]
	at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:732) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-455072c-43ab266]
	at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:400) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-455072c-43ab266]
	at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:668) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-455072c-43ab266]
	at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:567) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-455072c-43ab266]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_144]
Caused by: java.lang.NoSuchMethodError: org.bukkit.entity.Player.getWidth()D
	at ru.endlesscode.rpginventory.event.listener.ElytraListener.isPlayerCanFall(ElytraListener.java:62) ~[?:?]
	at ru.endlesscode.rpginventory.event.listener.ElytraListener.onPlayerFall(ElytraListener.java:47) ~[?:?]
	at sun.reflect.GeneratedMethodAccessor17.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_144]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_144]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:304) ~[spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-455072c-43ab266]
	... 15 more
```

**Comment by OsipXD** [comment]
Thanks for report, I'll fix it

**Comment by OsipXD** [comment]
Fixed in #80
